src/Symfony/Bundle/FrameworkBundle/Command/MessageConsumeCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Message\MessageBusInterface;
+use Symfony\Component\Message\Transport\ReceiverInterface;
+use Symfony\Component\Message\Worker;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class MessageConsumeCommand extends Command
+{
+    protected static $defaultName = 'message:consume';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDefinition(array(
+                new InputArgument('receiver', InputArgument::REQUIRED, 'Name of the receiver'),
+                new InputOption('bus', 'b', InputOption::VALUE_REQUIRED, 'Name of the bus to dispatch the messages to', 'message_bus'),
+            ))
+            ->setDescription('Consume a message')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command consume a message and dispatch it to the message bus.
```

Comment:
dispatches

Reply:
Maybe we should actually say "handles" or "dispatches to the registered handlers" to make it very explicit (following [this discussion](https://github.com/symfony/symfony/pull/24411#discussion_r160053724))